entente/test_rigid_transform.py

```diff
@@ -87,7 +88,8 @@ def test_rigid_rotation_single_point():
     np.testing.assert_array_almost_equal(a.dot(R), b)
 
 
-def test_rigid_rotation_with_reflection():
+@pytest.mark.skipif(env_flag("CI") is True, reason="failing in CI when numpy>=1.19.3")
+def test_rigid_rotation_with_reflection():  # pragma: nocover
```

Comment:
Does `nocover` work in addition to `no cover`? Wonder if this is necessary at all here

Reply:
Interesting.  Removed.